jylee0805/momoCall
# Feature/purchase

[Website URL](https://momocall--checkout-j5c8375e.web.app/chat?member)

## Description
- Complete guided checkout process
## Test Plan
- Click the link，and click 「立即購買」
- Choose the spec, add the the product num,click 「下一步」
- If product num is 0, can't turn to next page
- Type credit card number「4242 4242 4242 4242」，「01/25」、「123」，the order will complete when you click confirm
- If the credit card information has error，the error message will pop up
